module/enrichers.mjs

```diff
@@ -180,6 +180,77 @@ export async function enrichCheck(config, label, options) {
 
 /* -------------------------------------------- */
 
+/**
+ * Enrich a damage link.
+ * @param {string[]} config            Configuration data.
+ * @param {string} label               Optional label to replace default text.
+ * @param {EnrichmentOptions} options  Options provided to customize text enrichment.
```

Comment:
I might have missed this in the earlier review but these are documented as being optional but not marked in the JSDoc as optional, e.g.

```suggestion
 * @param {string} [label]               Optional label to replace default text.
 * @param {EnrichmentOptions} [options]  Options provided to customize text enrichment.
```

And that goes for `enrichCheck` too. I think `options` is guaranteed to be passed-in by the calling `TextEditor` methods, but since these functions are exported, presumably they might be called separately from that workflow. On that note, do we know why these methods are exported? It seems that the only entry-point we actually need is `registerCustomEnrichers`, and that the rest of the functions can be private to the module, no?